docs/changelog.md

```diff
@@ -239,31 +305,32 @@ sidebar_label: Changelog
   directory was scanned only upon startup. Now it is scanned each time the user
   requests to load a query module. The functions used to load the query modules
   were renamed to `mg.load()` and `mg.load_all()` (from `mg.reload()` and
-  `mg.reload_all()`).
-* Improved execution performance of queries that have an IN list filter by
-  using label+property indices.
-  Example: `MATCH (n:Label) WHERE n.property IN [] ...`
+  `mg.reload_all()` ).
+* Improved execution performance of queries that have an IN list filter by using
+  label+property indices. Example: `MATCH (n: Label) WHERE n.property IN [] ...`
+
 * Added support for `ANY` and `NONE` openCypher functions. Previously, only
   `ALL` and `SINGLE` functions were implemented.
 
 ### Bug Fixes and Other Changes
 
 * Fixed invalid paths returned by variable expansion when the starting node and
-  destination node used the same symbol.
-  Example: `MATCH path = (n:Person {name: "John"})-[:KNOWS*]->(n) RETURN path`
+  destination node used the same symbol. Example: `MATCH path = (n: Person
+  {name: "John"})-[: KNOWS*]->(n) RETURN path`
+
 * Improved semantics of `ALL` and `SINGLE` functions to be consistent with
-  openCypher when handling lists with `Null`s.
+  openCypher when handling lists with `Null` s.
 * `SHOW CONSTRAINT INFO` now returns property names as a list for unique
   constraints.
-* Escaped label/property/edgetype names in `DUMP DATABASE` to support names
-  with spaces in them.
-* Fixed handling of `DUMP DATABASE` queries in multi-command transactions
-  (`BEGIN`, ..., `COMMIT`).
+* Escaped label/property/edgetype names in `DUMP DATABASE` to support names with
+  spaces in them.
+* Fixed handling of `DUMP DATABASE` queries in multi-command transactions (
+  `BEGIN` , ..., `COMMIT` ).
```

Comment:
Are these extra spaces intentional ?

Reply:
I am not sure why everything has been formatted...

Reply:
No worries!